### ISSUE TITLE
How to detect PopUp Showing?

### ISSUE THREAD

**Issue by erolerme** [Tulpep/Notification-Popup-Window#15]
Hey how can i detect popup is showing now? Or can i restrict for only one popup object show at the same time?

Thank you..

**Comment by LiorBanai** [comment]
I forked the repo and added multi popup support if you are interested:
https://github.com/LiorBanai/Notification-Popup-Window

**Comment by ricardopolo** [comment]
@LiorBanai would you like to create a PR instead maintaining a new fork?

**Comment by LiorBanai** [comment]
Hi, @ricardopolo .
I  was under the impression that the repo is no longer active so I forked it to fix the left over issues and so unfortunately I have already changed a lot of stuff to the original source code.

**Comment by ricardopolo** [comment]
Ok @LiorBanai

**Comment by LiorBanai** [comment]
@ricardopolo  since I cannot double fork the repo, if you wish to give me push access I can make the fixes again directly to your repo via pr in the repo itself

**Comment by ricardopolo** [comment]
Yes, I would add you as contributor so you will be able to push directly :)
